src/server/room-controller.js

```diff
@@ -1,20 +1,53 @@
+import express from 'express';
```

Comment:
I implemented the controller according to what we agreed on in the start of the project (registration of endpoint and handler separated but in one file)

Reply:
I'll switch my stuff to this style as well when we merge. I totally forgot about it

Reply:
> I implemented the controller according to what we agreed on in the start of the project (registration of endpoint and handler separated but in one file)

I think so far this is the first such pattern.
Cool, let's start changing like this then 👍